src/Compilers/CSharp/Portable/CommandLine/CSharpCommandLineParser.cs

```diff
@@ -1193,12 +1193,9 @@ public new CSharpCommandLineArguments Parse(IEnumerable<string> args, string bas
                 keyFileSetting = ParseGenericPathToFile(keyFileSetting, diagnostics, baseDirectory);
             }
 
-            if (sourceLink != null)
+            if (sourceLink != null && !emitPdb)
```

Comment:
Yes, forgot about that.

---
In reply to: [113746766](https://github.com/dotnet/roslyn/pull/18539#discussion_r113746766) [](ancestors = 113746766)